### DISCUSSION TITLE
Issue while debugging

### DISCUSSION THREAD

**Discussion by Narayanhegde14** [FreeSK8/freesk8_mobile#22] [Q&A]
![image](https://user-images.githubusercontent.com/79844738/142727554-c680df02-4f68-4bec-95d8-e92645e3d378.png)

**Comment by r3n33** [comment]
I suspect Flutter needs to be updated. It looks like you are using version 1.5 and at the moment 2.5.3 is the latest stable version. 

FreeSK8 Mobile is compiled with the latest stable version of Flutter.

**Reply by Narayanhegde14** [reply]
I tried with 2.5.3 same error is repeating while debugging

**Reply by r3n33** [reply]
My apologies! Today I realized my IDE was using version 2.0.4 of Flutter. I pointed my IDE to the latest version and experience the same issue. I'll work on getting the dependencies updated. In the meantime you can use `flutter downgrade 2.0.4` to ensure a successful build

**Reply by Narayanhegde14** [reply]
I resolved that issue by upgrading all the packages which can be updated.
